src/requirements/data/colleges/en.ts

```diff
@@ -80,33 +97,47 @@ const engineeringRequirements: readonly CollegeOrMajorRequirement[] = [
     fulfilledBy: 'courses',
     minCount: 2,
   },
-  // TODO: Check for categories & handle changed requirements for those entering in 2020 and later
   {
     name: 'Liberal Studies Distribution: 6 courses',
     description:
       'Liberal arts commonly include courses in the humanities. A minimum of six courses must be taken. ' +
-      ' These courses must come from some different groups and at least two courses must be at the 2000 level or higher.',
+      'At least two courses must be at the 2000 level or higher.',
     source:
       'https://www.engineering.cornell.edu/students/undergraduate-students/advising/liberal-studies',
     checker: (course: Course): boolean =>
-      ['CA', 'HA', 'LA/LAD', 'KCM', 'SBA', 'FL', 'CE'].some(
+      engineeringLiberalArtsDistributions.some(
         distribution => course.catalogDistr?.includes(distribution) ?? false
       ),
     subRequirementProgress: 'any-can-count',
     fulfilledBy: 'courses',
     minCount: 6,
   },
+  {
+    name: 'Liberal Studies Distribution: 3 categories',
+    description:
+      'In addition to six courses, the liberal studies courses must be from 3 categories.',
+    source:
+      'https://www.engineering.cornell.edu/students/undergraduate-students/advising/liberal-studies',
+    checker: engineeringLiberalArtsDistributions.map(distribution => (course: Course): boolean =>
+      course.catalogDistr?.includes(distribution) ?? false
+    ),
+    subRequirementProgress: 'every-course-needed',
+    fulfilledBy: 'courses',
+    allowCourseDoubleCounting: true,
+    minCount: 3,
+  },
```

Comment:
also added this 3 category checker

Reply:
To confirm: This checker doesn't check if the courses fall under 3 distinct categories right?

Reply:
Just wanted to point out that it seems a single course that includes multiple distribution categories may "fulfill" this requirement. Is that ok? I believe that a post-launch goal is to be able to have category filters where the user can select a certain category (e.g. HA) for a course. So this 3 category subreq coupled with the other liberal arts subreqs may be ok for launch.

https://user-images.githubusercontent.com/54298311/108725903-d8054b80-74f4-11eb-80bc-9f045f72731f.mov

Reply:
also why are there add courses for up to 14? 
Is this because of the 13 categories?

Reply:
Yes, it's mostly for 3 categories. It's 14 instead of 13 due to LA vs LAD, but I can try to condense them into one to make it more intuitive.

Reply:
maybe for now, we could just count the first category a class fulfills for this req?
if we need to keep track of 3 separate categories, would this require knowing which categories the user already has in their schedule, checking if its new, and if so counting it?